### PR TITLE
New Package: sharkdp/bat/0.22.1

### PR DESCRIPTION
**Pull request by neomax7**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94719)

### CODE REVIEW

**Comment by neomax7** [comment]
manifest version 1.4 with zip

**Comment by neomax7** [comment]
@sitiom 
I have corrected the folder name :)

**Comment by neomax7** [comment]
v1.4 with zip

File: manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: sharkdp.bat
+PackageVersion: 0.22.1
+InstallerLocale: en-US
+InstallerType: zip
```
Comment (sitiom) [inline manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.installer.yaml:7]:
```suggestion
InstallerType: zip
ReleaseDate: 2022-09-11
```

File: manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PackageLocale: en-US
+Publisher: sharkdp
+PublisherUrl: https://github.com/sharkdp/bat
+PublisherSupportUrl: https://github.com/sharkdp/bat
+Author: sharkdp
+PackageName: bat
+PackageUrl: https://github.com/sharkdp/bat
+License: Apache-2.0
+LicenseUrl: https://github.com/sharkdp/bat
+ShortDescription: A cat(1) clone with syntax highlighting and Git integration.
```
Comment (sitiom) [inline manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.locale.en-US.yaml:15]:
```suggestion
Publisher: David Peter
PublisherUrl: https://david-peter.de/
Author: David Peter
PackageName: bat
PackageUrl: https://github.com/sharkdp/bat
License: Apache-2.0, MIT
LicenseUrl: https://github.com/sharkdp/bat#license
Copyright: Copyright (c) 2018-2021 bat-developers
CopyrightUrl: https://github.com/sharkdp/bat#license
ShortDescription: A cat(1) clone with syntax highlighting and Git integration.
Tags:
- cli
- rust
- cat
ReleaseNotesUrl: https://github.com/sharkdp/bat/releases/tag/v0.22.1
```

**Review by sitiom — CHANGES_REQUESTED** [review]

File: manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        PortableCommandAlias: bat
+    InstallerUrl: https://github.com/sharkdp/bat/releases/download/v0.22.1/bat-v0.22.1-x86_64-pc-windows-msvc.zip
+    InstallerSha256: D14028B3579DF7A5E1B85E5B565FFFD33B5E4096DF0C719F98F8381CE1DCCE6D
+  - Architecture: x86
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: bat-v0.22.1-i686-pc-windows-msvc\bat.exe
+        PortableCommandAlias: bat
+    InstallerUrl: https://github.com/sharkdp/bat/releases/download/v0.22.1/bat-v0.22.1-i686-pc-windows-msvc.zip
+    InstallerSha256: 7B66CD89DDDF6872164CD925D4C26A14D4A7DBB6A98983EB8542677CD823358E
```
Comment (sitiom) [inline manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.installer.yaml:23]:
```suggestion
    InstallerSha256: 7B66CD89DDDF6872164CD925D4C26A14D4A7DBB6A98983EB8542677CD823358E
    Dependencies:
      PackageDependencies:
        - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+InstallerType: zip
+ReleaseDate: 2022-09-11
+Installers:
+  - Architecture: x64
+    NestedInstallerType: portable
+    NestedInstallerFiles:
+      - RelativeFilePath: bat-v0.22.1-x86_64-pc-windows-msvc\bat.exe
+        PortableCommandAlias: bat
+    InstallerUrl: https://github.com/sharkdp/bat/releases/download/v0.22.1/bat-v0.22.1-x86_64-pc-windows-msvc.zip
+    InstallerSha256: D14028B3579DF7A5E1B85E5B565FFFD33B5E4096DF0C719F98F8381CE1DCCE6D
```
Comment (sitiom) [inline manifests/s/sharkdp/bat/0.22.1/sharkdp.bat.installer.yaml:16]:
```suggestion
    InstallerSha256: D14028B3579DF7A5E1B85E5B565FFFD33B5E4096DF0C719F98F8381CE1DCCE6D
    Dependencies:
      PackageDependencies:
        - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by sitiom — CHANGES_REQUESTED** [review]
Requires VCRedist dependency

**Comment by sitiom** [comment]
@neomax7 Try closing and reopening all your open PRs so that the validation pipelines are rerun.

**Pull request closed without merging**